app.go

```diff
@@ -3,20 +3,22 @@ package main
 import (
 	"bytes"
 	"flag"
+	"github.com/cloudogu/carp"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"strconv"
 
 	"fmt"
 
-	"github.com/cloudogu/carp"
 	"github.com/cloudogu/go-health"
-	"github.com/golang/glog"
+	logging "github.com/op/go-logging"
 )
 
 var Version = "x.y.z-dev"
 
+var log *logging.Logger
```

Comment:
```suggestion
var log *logging.Logger = logging.MustGetLogger("nexus-carp")
```
Initialize the logger with default settings to use it for log messages which are written when the configuration could not be loaded.